### ISSUE TITLE
Evaluating with an invalid key raises a 500

### ISSUE THREAD

**Issue by rcrowe** [configcat/configcat-proxy#18]
### Describe the bug

If an invalid key is supplied a 500 internal error along with text response (text/plain) is returned.
I'm classing an invalid key as missing or empty, haven't explored alternatives.

### To reproduce

Either no key:

```shell
curl -X "POST" "http://localhost:8050/api/{sdkId}/eval" \
     -H 'Content-Type: application/json; charset=utf-8' \
     -d $'{}'
```

Or an empty key:

```shell
curl -X "POST" "http://localhost:8050/api/{sdkId}/eval" \
     -H 'Content-Type: application/json; charset=utf-8' \
     -d $'{
  "key": ""
}'
```

### Expected behavior

A more appropriate HTTP status code that indicates a client error (400).

**Comment by z4kn4fein** [comment]
Hi @rcrowe, thank you for reporting this issue! I will check this and let you know when the fix is ready.

**Comment by z4kn4fein** [comment]
Hi @rcrowe, I've fixed this issue in `v0.4.1`. Could you please check that it works now as you expected? Thanks!
